File: server/src/main/java/com/example/server/note/domain/Markdown.java
package com.example.server.note.domain;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@Getter
@Setter
@Entity
public class Markdown extends Date {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @Column(nullable = false)
    private String slug;

    @Column(columnDefinition = "TEXT")
    private String content;


}

File: server/src/main/kotlin/work/notespace/server/note/entity/Markdown.kt
package work.notespace.server.note.entity

import jakarta.persistence.*
import work.notespace.server.note.dto.MarkdownDto


@Entity
class Markdown(
    @Id
    @Column
    @GeneratedValue(strategy = GenerationType.AUTO)
    val id: Long? = null,

    @Column
    val slug: String,

    @Column
    var content: String = "",
) : Date() {
    fun toDto(): MarkdownDto {
        return MarkdownDto(
            id = id,
            slug = slug,
            content = content,
        )
    }
}